src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price;
+  const rent = 40;
+
+  if (days >= 1) {
+    price = days * rent;
+  }
+
+  if (days >= 3) {
+    price = (days * rent) - 20;
```

Comment:
not critical, but you can move 20 and 50 to variables with meaningful names to make it clearer